packages/yew/src/utils/mod.rs

```diff
@@ -95,6 +95,12 @@ impl<IN, OUT> IntoIterator for NodeSeq<IN, OUT> {
     }
 }
 
+/// Hack to force type mismatch compile errors in yew-macro.
+//
+// TODO: replace with `compile_error!`, when `type_name_of_val` is stabilised.
```

Comment:
```suggestion
// TODO: replace with `compile_error!`, when `type_name_of_val` is stabilised (https://github.com/rust-lang/rust/issues/66359).
```
I'm always tempted by an issue link to go check out if something has changed :D